test/functional/DDR_Test/playlist.xml

```diff
@@ -14,18 +14,17 @@ forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
 General Public License, version 2 with the GNU Classpath
 Exception [1] and GNU General Public License, version 2 with the
 OpenJDK Assembly Exception [2].
-
```

Comment:
Please restore the blank lines that separate paragraphs of the copyright notice.